src/sentry/migrations/0855_update_group_open_periods_constraint.py

```diff
@@ -0,0 +1,52 @@
+# Generated by Django 5.1.7 on 2025-04-01 20:22
+
+import django.contrib.postgres.constraints
+import django.contrib.postgres.fields.ranges
+from django.db import migrations, models
+
+import sentry.models.groupopenperiod
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production.
+    # This should only be used for operations where it's safe to run the migration after your
+    # code has deployed. So this should not be used for most operations that alter the schema
+    # of a table.
+    # Here are some things that make sense to mark as post deployment:
+    # - Large data migrations. Typically we want these to be run manually so that they can be
+    #   monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   run this outside deployments so that we don't block them. Note that while adding an index
+    #   is a schema change, it's completely safe to run the operation after the code has deployed.
+    # Once deployed, run these manually via: https://develop.sentry.dev/database-migrations/#migration-deployment
+
+    is_post_deployment = False
+
+    dependencies = [
+        ("sentry", "0854_add_project_sdk_model"),
+    ]
+
+    operations = [
+        migrations.RemoveConstraint(
+            model_name="groupopenperiod",
```

Comment:
This table is empty right? The migration framework is complaining that this is dangerous, but we can just set `checked=False` on the migration class to skip the check since we know this is safe.

Reply:
ty, added